### DISCUSSION TITLE
is there smtp_mail_reply_to config

### DISCUSSION THREAD

**Discussion by karunpoudel** [apache/airflow#25917] [Q&A]
Similar to [smtp_mail_from](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#smtp-mail-from) config, is there smtp_mail_reply_to config?
Our mail_from value is like airflow_dev@company.com, airflow_prod@company.com but we want the reply-to to be correct team email address.

**Comment by potiuk** [comment]
No. But it should be rather easy to contribute it back - so feel free to do so and join more than 2100 contributors to airflow. This is an easy way to pay back for the free software you get.
